external/fv3fit/fv3fit/pytorch/building_graph.py

```diff
@@ -0,0 +1,52 @@
+import torch
```

Comment:
Please rename this file to be a noun instead of a verb.

I notice you have many files named `graph_<something>`. It may make sense to collect these under a submodule, i.e. create a "graph" directory with an `__init__.py` file and put these modules in there as e.g. graph.config, graph.loss, etc.